lib/internal/process/promises.js

```diff
@@ -41,9 +50,12 @@ function setupPromises(scheduleMicrotasks) {
       var reason = pendingUnhandledRejections.shift();
       if (hasBeenNotifiedProperty.get(promise) === false) {
         hasBeenNotifiedProperty.set(promise, true);
+        const uid = promiseToGuidProperty.get(promise);
```

Comment:
`promiseToGuidProperty.get(promise)` uses also [here](https://github.com/nodejs/node/pull/6439/files#diff-abbefee1c196cd57720161c414eaaa96R31), maybe worth making it a method instead?